src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
```

Comment:
The Main class should not start with an empty line. Please remove the empty line at the beginning of the class implementation.